src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

Comment:
We met with reviews to better add something like this 

transition: transform 300ms, color 300ms;

than default 'all'

Reply:
support @proznerka 
using `all` downside is it slows down load times